shell/packages/sandstorm-ui-powerbox/powerbox.html

```diff
@@ -0,0 +1,80 @@
+<template name="powerboxRequest">
+  <h4>Powerbox request</h4>
+  {{#if showWebkeyInput}}
+    <form id="powerbox-request-form">
+      <label>Please input an api token: </label><input name="token" type="text" id="powerbox-request-input"><br>
+      <button class="submit">Import token</button>
+    </form>
+    {{#if webkeyError}}
+    <div class="error">{{webkeyError}}</div>
+    {{/if}}
+  {{else}}
+    <h4>Connect this grain with:</h4>
```

Comment:
FWIW, I think we can reasonably have different language when selecting SealedUiView vs. other capability types. "Connect" could make more sense for general capabilities, maybe.

"Select one:" also seems reasonable.